src/backend/distributed/metadata/node_metadata.c

```diff
@@ -1160,6 +1163,17 @@ ActivateNodeList(MetadataSyncContext *context)
 }
 
 
+/*
+ * Adds locks into all shards placed into given workerNode.
```

Comment:
`Add lock` is an unfamiliar term. Maybe `Acquire shard metadata locks on all shards residing in the given worker node`

Reply:
also, this should definitely state something like the following (probably with better wording):
```
TODO: This function is not compatible with query from any node feature. Acquiring locks on placements should happen on all nodes, not only on the coordinator (or more precisely the node this function called)
```

Reply:
Added